dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -67,6 +57,33 @@
 		<_ProjectType Condition="'$(_ProjectType)' == '' And '$(_PlatformName)' == 'macOS' And '$(OutputType)' == 'Library'">macOSClassLibrary</_ProjectType>
 	</PropertyGroup>
 
+	<!-- Automatically supply project capabilities for IDE use -->
+	<ItemGroup>
+		<ProjectCapability Include="Apple" />
+		<ProjectCapability Include="Mobile" />
+		
+		<ProjectCapability Include="iOSApplication" Condition="'$(_ProjectType)' == 'iOSExecutableProject'" />
+		<ProjectCapability Include="iOSAppExtension" Condition="'$(_ProjectType)' == 'iOSAppExtensionProject'" />
+		<ProjectCapability Include="iOSBinding" Condition="'$(_ProjectType)' == 'iOSBindingProject'" />
+		<ProjectCapability Include="iOSClassLibrary" Condition="'$(_ProjectType)' == 'iOSClassLibrary'" />
+		<ProjectCapability Include="TvOSApplication" Condition="'$(_ProjectType)' == 'tvOSExecutableProject'" />
```

Comment:
There's a capitalization mismatch here: "iOS" starts with a lower cased i, while all the other start with an uppercased letter. Could this be made consistent (either way is fine, although I tend to prefer Apple's casing with the lowercased first letter)?

Reply:
@rolfbjarne I have replaced the `i` for an `I`, as far as I can see on project capabilities doc all are in upper case.